docs/guide/cli/index.md

```diff
@@ -0,0 +1,253 @@
+# CLI
+
+The e18e CLI is a powerful tool for analyzing and optimizing your JavaScript/TypeScript projects. It helps you identify performance issues, find optimization opportunities, and migrate to better alternatives.
+
+## Quick Start
+
+```bash
+# Install and run analysis
+npx @e18e/cli analyze
+
+# Migrate packages interactively
+npx @e18e/cli migrate --interactive
+```
+
+## Installation
+
+You can install the e18e CLI globally or use it with npx:
+
+::: code-group
+
+```sh [npm]
+npm install -g @e18e/cli
+```
+
+```sh [yarn]
+yarn global add @e18e/cli
+```
+
+```sh [pnpm]
+pnpm add -g @e18e/cli
+```
+
+```sh [bun]
+bun add -g @e18e/cli
+```
+
+:::
+
+## Commands
+
+### `analyze`
+
+Analyzes your project for various issues and provides recommendations for optimization.
+
+```bash
+e18e-cli analyze [path] [options]
```

Comment:
these examples should be `npx @e18e/cli analyze [path] [options]` i think

same for the other ones, the name would be `@e18e/cli`